### DISCUSSION TITLE
Error saat dirun: Expected an object with class 'shiny.tag'.

### DISCUSSION THREAD

**Discussion by michelleintanh** [teamalgoritma/community#682] [Q&A] [answered]
[capstone_michelle.zip](https://github.com/teamalgoritma/community/files/11651810/capstone_michelle.zip)

Bukti error:
<img width="675" alt="image" src="https://github.com/teamalgoritma/community/assets/127676132/1a5c5096-c34d-4036-9b4c-57a939ced878">

**Comment by jafarhf1** [comment, marked as answer]
Dear @michelleintanh ,

Terima kasih atas pertanyaannya, setelah kami telusuri berdasarkan file yang telah dilampirkan, kendalanya adalah masalah peletakan tanda kurung. Silahkan dicek kembali mengenai posisi kurungnya. Salah satunya misalnya adalah semua penggunaan `tabItem()` adalah harus didalam `tabItems()`

Demikian yang dapat kami sampaikan. Semoga membantu

**Reply by michelleintanh** [reply]
Baik, terima kasih kak.
